nailgun/entities.py

```diff
@@ -1830,7 +1843,9 @@ def read(self, entity=None, attrs=None, ignore=None):
                 self._server_config,
                 id=attrs['id']
             ).update_json([])
-            for attr in ('content_view_id', 'lifecycle_environment_id'):
+            for attr in ('content_view_id',
+                         'lifecycle_environment_id',
+                         'content_source_id'):
```

Comment:
Aha. So this attribute is not returned in response to either `POST` or `GET` requests. :cry: